plugins/snippets/nvim-snippets.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "nvim-snippets";
+  luaName = "snippets";
+  defaultPackage = pkgs.vimPlugins.nvim-snippets;
+
+  maintainers = [ lib.maintainers.psfloyd ];
+
+  settingsOptions = with helpers.nixvimTypes; {
+    create_autocmd = helpers.defaultNullOpts.mkBool false ''
+      Optionally load all snippets when opening a file.
+      Only needed if not using nvim-cmp.
+    '';
+    create_cmp_source = helpers.defaultNullOpts.mkBool true ''
```

```suggestion
    '';

    create_cmp_source = helpers.defaultNullOpts.mkBool true ''
```

Please, add a line break between each option declaration.